src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -1332,15 +1360,16 @@ public virtual string ProxyPassword
                 if (!_proxyPasswordEvaluated)
                 {
                     var hasObscuringKey = !string.IsNullOrWhiteSpace(ObscuringKey);
```

Comment:
It's worth noting, I think, that the obscuring key can also be configured via the env var `NEW_RELIC_CONFIG_OBSCURING_KEY`.  In our config docs, we currently tell users to put the obscuring key in the environment and the obscured values in the config file (otherwise the obfuscation is almost pointless).   We might need to update that statement when we update the config docs with these new env vars.

Reply:
Yeah, I will leave this here so I remember when working on docs